controller/src/utils.ts

```diff
@@ -1,11 +1,65 @@
 import equal from "fast-deep-equal";
-
 import { Policy } from "./types";
+import { GraphQLClient } from "graphql-request";
+import { number, hash, shortString } from "starknet";
+import BN from "bn.js";
+
+export const ENDPOINT = process.env.NEXT_PUBLIC_API_URL!;
+
+export const client = new GraphQLClient(ENDPOINT, { credentials: "include" });
+
+export const PROXY_CLASS =
+  "0x04be79b3904b4e2775fd706fa037610b41d8f8708ce298aac3a470badf68176d";
+
+export const CLASS_HASHES = {
```

Comment:
can we move this to a `constants` file and have keychain import it from controller to avoid duplicating?